### PR TITLE
Add support for EuroLinux

### PR DESCRIPTION
**Pull request by AlexBaranowski**
## Description

Add support for EuroLinux distro.

### CODE REVIEW

**Comment by dylanaraps** [comment]
Thanks for the PR :+1:. Any chance at a slightly smaller logo? It's a little large!

**Comment by AlexBaranowski** [comment]
Done :smile:.

**Comment by dylanaraps** [comment]
LGTM :+1: Thanks!

File: neofetch [hunk 1]
```diff
@@ -6268,6 +6268,32 @@ dMm     `/++/-``/yNNh+/sdNMNddMm-    mMd
 EOF
         ;;
 
+        "EuroLinux"*)
+            set_colors 4 7
+            read -rd '' ascii_data <<'EOF'
+
```
Comment (dylanaraps) [inline neofetch]: Remove this line. :+1:

**Comment by AlexBaranowski** [comment]
I hope that this is what you thought.

**Comment by dylanaraps** [comment]
Thanks!

**Pull request merged**